fcrepo4/fcrepo4
# Adds a "<> rdf:type memento:TimeMap" triple to the body of TimeMap (L…

**Adds a "<> rdf:type memento:TimeMap" triple to the body of TimeMap (LDPCv - /fcr:versions)  resource.**
* * *

**JIRA Ticket**: https://jira.duraspace.org/browse/FCREPO-3006

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?
It simply adds the aforementioned triple to the timemap body by adding the required rdf:type to the jcr model via the mixin mechanism. It includes an integration test.

# How should this be tested?

Create a resource
```
curl -u fedoraAdmin:fedoraAdmin http://localhost:8080/rest/test -X PUT
```
GET the timemap
```
curl -u fedoraAdmin:fedoraAdmin http://localhost:8080/rest/test/fcr:versions -X GET
```
Verify the `<> rdf:type memento:TimeMap` triple is present in the response.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

Example:
* Does this change require documentation to be updated? No
* Does this change add any new dependencies?  No
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)?  No
* Could this change impact execution of existing code? No

# Interested parties
Tag (@ mention) interested parties or, if unsure, @fcrepo4/committers
